examples/examples/z_ping.rs

```
@@ -123,6 +126,13 @@ fn parse_args() -> (Config, Duration, usize, usize) {
     if args.is_present("no-multicast-scouting") {
         config.scouting.multicast.set_enabled(Some(false)).unwrap();
     }
+    if args.is_present("enable-shm") {
+        #[cfg(feature = "shared-memory")]
+        config.transport.shared_memory.set_enabled(true).unwrap();
+        #[cfg(not(feature = "shared-memory"))]
+        warn!("enable-shm argument: SHM cannot be enabled, because Zenoh is compiled without shared-memory feature!");
```

I don't think logging is needed. Just print and exit. Let the application fail.